manifests/sysstat.pp

```diff
@@ -0,0 +1,24 @@
+# === Copyright
+#
+# Copyright 2014, Deutsche Telekom AG
+# Licensed under the Apache License, Version 2.0 (the "License");
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# == Class: os_hardening::sysstat
+#
+# Configures /etc/default/sysstat
+#
+class os_hardening::sysstat (
+  $sysstat_enabled = 'true',
+){
+  # set the file
+  file {
+    '/etc/default/sysstat':
```

Comment:
@artem-sidorenko Unfortunately, I am not aware of the centos/rhel. However, this file is modified by setting "ENABLED = true" which is Should sadc collect system activity information's?.